src/RHevent_structure.py

```diff
@@ -0,0 +1,91 @@
+#
+# Event Structure
+# Provides methods for running race events and event formats
+#
+
+class EventStructure():
+    
+    def generate_bracket_positions(self, num_brackets, slots, participants):
```

Comment:
This whole file actually needs to go away. It was an early experiment and isn't called by anything. I haven't removed it yet because it contains the only work I've done toward parameterized bracket generation—which I do want to add later on. No surprise it's still quite a mess.